imsc1/spec/ttml-ww-profiles.html

```diff
@@ -4609,34 +4611,62 @@ <h2>WCAG and MAUR Considerations</h2>
+        cannot see images (the <a>non-text content</a>). Since the images of an <a>Image Profile</a> <a>Document Instance</a>
+        usually represent subtitle or caption text, the guidelines for authoring text equivalent strings given at <span
+        class='sec-no'>Images of text</span> of [[HTML5]] are appropriate.</p>
+
+        <p>Thus, for each subtitle in an <a>Image Profile</a> <a>Document Instance</a>, a text equivalent content in a <a>Text
+        Profile</a> <a>Document Instance</a> SHOULD be written so that it conveys all essential content and fulfills the same
+        function as the corresponding subtitle image. In the context of subtitling and captioning, this content will be (as a
+        minimum) the verbatim equivalent of the image without précis or summarization. However, the author MAY include extra
+        information to the text equivalent string in cases where styling is applied to the text image with a deliberate connotation,
+        as a <em>functional</em> replacement for the applied style.</p>
+
+        <aside class="example">In subtitling and captioning, italics can be used to indicate an off screen speaker context (for
+        example a voice from a radio). An author can choose to include this functional information in the text equivalent; for
+        example, by including the word "Radio: " before the image equivalent text. Note that images in an <a>Image Profile</a>
+        <a>Document Instance</a> that are intended for use as <em>captions</em>, i.e. intended for a hard of hearing audience, might
+        already include this functional information in the rendered text.</aside>
+      </section>
+
+      <section class='appendix' id='wcag-criterion-contrast'>
+        <h4>Success Criterion 1.4.3 Contrast (Minimum)</h4>
+      
+        <p><a data-cite="!WCAG21#contrast-minimum">Success Criterion 1.4.3</a> at [[!WCAG21]] specifies that, with some exceptions,
+        the visual presentation of text and images of text has a contrast ratio of at least 4.5:1.</p>
+
+        <p>In the context of <a>Text Profile</a>, this is achieved by specifying appropriate foreground and background colors.</p>
+      </section>
+
+      <section class='appendix' id='wcag-criterion-text=spacing'>
+        <h4>Success Criterion 1.4.12 Text Spacing</h4>
 			
-			<p><a data-cite="!WCAG21#text-spacing">Guideline 1.4.12</a> of [[!WCAG21]] also recommends that no loss of content or functionality occur when spacing of lines, letters, words and paragraphs is modified by a specified amount.</p>
+        <p><a data-cite="!WCAG21#text-spacing">Success Criterion 1.4.12</a> at [[!WCAG21]] specifies that no loss of content or
+          functionality occur when spacing of lines, letters, words and paragraphs is modified by a specified amount.</p>
+
+        <p>In the context of this specification, this is achieved by specifying an appropriate combination of font and text styles,

```

Comment:
What is a "font style" here as opposed to a "text style"? I'm not sure if this means attributes applied to a `<font>` element or if it is making a distinction between style (`tts:`) attributes that affect font vs other style attributes?

Reply:
> if it is making a distinction between style (tts:) attributes that affect font vs other style attributes?

Yes.

Reply:
> Yes.

Thanks, in that case:

```suggestion
        <p>In the context of this specification, this is achieved by applying an appropriate combination of styles to layout and content elements to achieve the desired
```